backfill.sql

```diff
@@ -287,7 +309,13 @@ BEGIN
     COMMIT;
 
     --Move our job back to where it was
-    SELECT move_compression_job(hypertable_row.id, hypertable_row.schema_name, hypertable_row.table_name, old_compression_job_time) INTO old_compression_job_time;
+    SELECT
+        move_compression_job(
+            hypertable_row.id,
+            hypertable_row.schema_name,
+            hypertable_row.table_name,
+            old_compression_job_time
+        ) INTO old_compression_job_time;
```

Comment:
We prefer to send separated PRs for code formatting because we can add it to .git-blame-ignore-revs

Reply:
@coiax Could you revert this formatting change as it's unrelated?